src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -2758,7 +2796,8 @@ public enum ElementState
         Warning,
         PersistentWarning,
         Error,
-        AstBuildBroken
+        AstBuildBroken,
+        Info
```

Comment:
@QilongTang @Amoursol @reddyashish I was thinking if we could reuse `warning` or `error` for integer overflow notifications, or is there a reason we are going for a new state `Info` here? Just curious.

Reply:
@zeusongit Error is only used when the thing, such as a graph, or code block will not run (i.e. missing a package so you have dummy nodes). Warning is when something has occured that "may cause your graph to execute in a way you do not expect", but it is up to the end user to determine that. The new "Info" state is purely informational, a kind of FYI.

We added the info state for many reasons, but one strong one is the concern users have around warnings, thinking their graph has failed. This is especially problematic in Dynamo Player (Which surfaces a message stating "Your graph has ran with warnings") as the end users there may have on conception of Dynamo, what a node is, what an error is etc.

Reply:
Gotcha, thanks 👍🏽

Reply:
If this property is serialized to the dyn, it can break deserialization, @mjkkirschner?

Reply:
Will this be break? I think reordering the elements in a enum will be a break. Not sure.

Reply:
I don't think we serialize ElementState on the nodeModel - simply adding a new value to the enum is not an issue if we don't actually serialize and deserialize (as long as that enum is added to the end of the enum - no reordering allowed)

so in conclusion - 

make sure NodeModel.State is not serialized - or property that is of type `ElementState`